libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.estimateMaxSpendable.integration.test.ts

```diff
@@ -26,7 +26,7 @@ describe("testing estimateMaxSpendable", () => {
   });
 
   it("should estimate max spendable correctly", async () => {
-    await wallet.syncAccount(account);
+    await wallet.syncAccount(account, 1000000);
```

Comment:
Minor but I think we should run a full sync in integration tests

Reply:
fixed